lib/tracer/tracer_store.js

```diff
@@ -2,7 +2,9 @@ import { _ } from 'meteor/underscore';
 import { EJSON } from 'meteor/ejson';
 let logger = Npm.require('debug')('kadira:ts');
 
-TracerStore = function TracerStore (options) {
+export let TracerStore;
+
+TracerStore = function (options) {
```

Comment:
Could these lines be combined for consistency with how it is done in other files?